### ISSUE TITLE
Text editor in default theme is hard to see

### ISSUE THREAD

**Issue by JunaidQadirB** [mAAdhaTTah/WP-Gistpen#221]
After installing and activating the plug-in when I try to create a new gist, I cannot see a text editor where I can paste/type my code. 
I'm using 

- WordPress Version 4.9.5

- GistPen Version 1.0.2

Please see the screenshot.

![screenshot](https://user-images.githubusercontent.com/642353/38782166-65224276-4108-11e8-8ac6-e2447d4a20d7.png)

**Comment by mAAdhaTTah** [comment]
The screenshot actually shows the editor, although from a UX perspective, it's hard to tell. You can click on textbox (which again, is hard to see) to start writing in the editor.

**Comment by mAAdhaTTah** [comment]
If you change `Theme: Default` to something else (e.g. one of the dark themes), you should be able to see it a little clearer.

**Comment by JunaidQadirB** [comment]
I'm using firefox. For me its not working.

![gistpin-issue](https://user-images.githubusercontent.com/642353/38782518-2638cd72-410e-11e8-9242-168855c3180a.gif)

**Comment by mAAdhaTTah** [comment]
I clearly have some UX work to do. The dot you clicked on is for the filename. The rest of the black area is the editor.

**Comment by mAAdhaTTah** [comment]
This should be resolved by the changes coming in the next version.
